content/chapters/app-interact/lab/content/os-cloud.md

```diff
@@ -3,6 +3,27 @@
 In this section we are going to build a "toy cloud" called `OS Cloud`.
 Similar to a real cloud (like `aws`), `OS Cloud` will allow us to create and manage virtual machines, through an `http` API.
 
+## Containers vs VMs
+
+Containers are a lightweight virtualization technology that allows multiple isolated user-space instances to run on a single host operating system.
+They are often compared to chroots because they both provide isolated environments for running applications.
```

Comment:
```suggestion
They are often compared to [`chroot`](https://linux.die.net/man/1/chroot) because they both provide isolated environments for running applications.
```